packages/sanity/src/core/studio/components/navbar/presence/PresenceMenu.tsx

```diff
@@ -98,16 +98,23 @@ export function PresenceMenu(props: PresenceMenuProps) {
       menu={
         <StyledMenu padding={1}>
           {hasPresence && (
-            <Stack space={2}>
-              {presence.map((item) => (
-                <PresenceMenuItem
-                  focused={focusedId === item.user.id}
-                  key={item.user.id}
-                  onFocus={handleItemFocus}
-                  presence={item}
-                />
-              ))}
-            </Stack>
+            <>
+              <Box margin={2}>
```

Comment:
I suggest updating the JSX of this header slightly to use this:

```ts
<Box paddingX={3} paddingY={3}>
  <Label size={1} muted>
    See where they are
  </Label>
</Box>
```

This would also better match the design we have in the workspaces menu! (cc: @mikolajdobrucki)

<img width="928" alt="Screenshot 2023-02-03 at 14 03 13" src="https://user-images.githubusercontent.com/15094168/216610332-a5cdcdf4-6e38-4439-a038-62098bfde566.png">

Reply:
great catch @hermanwikner, the right version looks much better

Reply:
Yes, that makes sense, I didn't think about `Label`, but that seems to be more in line with the Figma as well